azurelinuxagent/ga/update.py

```diff
@@ -829,6 +831,24 @@ def _ensure_extension_telemetry_state_configured_properly(protocol):
         except Exception as e:
             logger.warn("Error when trying to delete existing Extension events directory. Error: {0}".format(ustr(e)))
 
+    @staticmethod
+    def _ensure_firewall_rules_persisted(dst_ip):
+        is_success = False
+        try:
+            PersistFirewallRulesHandler(dst_ip=dst_ip, uid=os.getuid()).setup()
+            msg = "Persistent firewall rules added successfully"
```

Comment:
can we rephrase this message and the one at 843... they give the impression that the rules were added, when they were actually just setup... maybe add "setup" somewhere in the message?

Reply:
Technically they were added (but not applied :) ) 
But yeah fair point, rephrased the message